docs/src/guides/themes.md

```diff
@@ -119,7 +119,7 @@ the right (but hey, you're free to do so!).
 The following elements support alignment:
 * Code blocks.
 * Slide titles.
-* The title, subtitle, and author elements in the intro slide.
+* The title, subtitle, event, location, date, and the author elements in the intro slide.
```

Comment:
I appreciate the effort but I tend to not document new features until they're released. Otherwise people may try to use them using the latest version and be surprised they're not available. This could be set up in some form where you have the nightly/latest docs separately but it's not set up/don't know if it's worth it yet so I'd prefer to keep this undocumented until it's released.